index.html

```diff
@@ -0,0 +1,75 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=  device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="navbar">
+        <header class="first-header">
+        Technical Documentation
+        </header>
+        <ul>
+            <li><a href="#The_first_section" class="nav-link">The first section</a></li>
+            <li><a href="#Introduction" class="nav-link">Introduction</a></li>
+            <li><a href="#What_you_Should_know" class="nav-link">What you should already know?</a></li>
+            <li><a href="#what_is_Javascript_and_Java?" class="nav-link">What is Javascript and Java?</a></li>
+            <li><a href="#Hello_World!" class="nav-link">Hello World!</a></li>
+        </ul>
+    </nav>
```

Comment:
I think the navbar should be inside the header. It will make your webpage more semantically correct.

Reply:
OPTIONAL: Well-written review comment :clap:. If you use emojis and/or gifs your comments would become more human friendly :robot: :alien: :man_astronaut:.